content/version/2/0/code_of_conduct.es.md

```diff
@@ -0,0 +1,84 @@
++++
+version = "2.0"
+aliases = ["/version/2/0/es"]
++++
+
+# Código de Conducta convenido para Contribuyentes
+
+## Nuestro compromiso
+
+Nosotros, como miembros, contribuyentes y administradores nos comprometemos a hacer de la participación en nuestra comunidad una experiencia libre de acoso para todo el mundo, independientemente de la edad, dimensión corporal, minusvalía visible o invisible, etnicidad, características sexuales, identidad y expresión de género, nivel de experiencia, educación, nivel socio-económico, nacionalidad, apariencia personal, raza, religión, o identidad u orientación sexual.
+
+Nos comprometemos a actuar e interactuar de maneras que contribuyan a una comunidad abierta, acogedora, diversa, inclusiva y sana.
+
+## Nuestros estándares
+
+Ejemplos de comportamiento que contribuyen a crear un ambiente positivo para nuestra comunidad:
+
+* Demostrar empatía y amabilidad ante otras personas
+* Respeto a diferentes opiniones, puntos de vista y experiencias
+* Dar y aceptar adecuadamente retroalimentación constructiva
+* Aceptar la responsabilidad y disculparse ante quienes se vean afectados por nuestros errores, aprendiendo de la experiencia
+* Centrarse en lo que sea mejor no sólo para nosotros como individuos, sino para la comunidad en general
+
+Ejemplos de comportamiento inaceptable:
+
+* El uso de lenguaje o imágenes sexualizadas, y aproximaciones o atenciones sexuales de cualquier tipo
+* Comentarios despectivos (_trolling_), insultantes o derogatorios, y ataques personales o políticos
+* El acoso en público o privado
+* Publicar información privada de otras personas, tales como direcciones físicas o de e-mail, sin su permiso explícito
```

Comment:
This may be vague but I think a non-technical term would fit better

```suggestion
* Publicar información privada de otras personas, tales como direcciones físicas o de correo electrónico, sin su permiso explícito
```

Reply:
Hm, you're right. I thought that _e-mail_ was already such an ubiquous term that it wouldn't matter, but now that I'm looking to the 1.4 translation, it also uses _correo electrónico_, so I'll make the switch.

(Plus, language academics are always saying that if you can choose between an English word or its accepted Spanish translation, the latter should be preferred).